routes/index.js

```diff
@@ -24,8 +24,8 @@ router.post("/api/nuevo", function (req, res, next) {
   console.log("--> ", req.body);
```

Comment:
Please, remove this and all the console logs in the code, but those that tells status of the db conection and port where is runnig the serve

Reply:
Done!